build.sbt

```diff
@@ -185,6 +191,44 @@ lazy val httpTests = project("akka-http-tests")
   .addAkkaModuleDependency("akka-actor-typed", "provided")
   .addAkkaModuleDependency("akka-multi-node-testkit", "test")
 
+lazy val copyDeps = taskKey[Unit]("Copies all dependencies in a special directory")
+
+lazy val osgiTest = taskKey[Unit]("Executes OSGi integration tests")
+
+lazy val httpOsgiTests = project("akka-http-osgi-tests")
+  .settings(Dependencies.httpOsgiTests)
+  .settings(
+    fork := true,
+    copyDeps := {
+      val httpFile = (http / Compile / OsgiKeys.bundle).value
```

Comment:
This is quite a bit of logic, I wonder if it'd make sense to move it to `project/OSGiTesting.scala`.

Reply:
As I said, there is certainly much room for improvement. You guys know for sure better how logic can be organized in an sbt build so that it can be easier maintained.